src/app/projects/add-edit-project/add-edit-project.component.scss

```diff
@@ -45,3 +45,7 @@
   font-size: 0.75rem;
   font-weight: normal;
 }
+
+#logos {
```

Comment:
might want to consider using classes, I'm not a big fan of using ids, especially ones, that sound like you could use it all over the place.  a good example of an id might be. #primary-navigation (even that could be argued).  Maybe future considerations... No response required ;)